docs/reference/api/docker_remote_api_v1.20.md

```diff
@@ -140,9 +140,14 @@ Create a container
                    "com.example.license": "GPL",
                    "com.example.version": "1.0"
            },
-           "Volumes": {
-                   "/tmp": {}
-           },
+           "Mounts": [
+             {
+               "Source": "/data",
+               "Destination": "/data",
+               "Relabel": "ro,Z",
```

Comment:
Right, this is a mode and SELinux will parse it correctly to figure out whether or not it needs to relabel.  Calling it relabel makes you think it is a boolean.